src/js/views/maps/ViewfinderView.js

```diff
@@ -0,0 +1,154 @@
+"use strict";
+
+define([
+  "backbone",
+  "text!templates/maps/viewfinder.html",
+], (
+  Backbone,
+  Template,
+) => {
+  /**
+   * @class ViewfinderView
+   * @classdesc The ViewfinderView allows a user to search for a latitude and longitude in the map view.
+   * @classcategory Views/Maps
+   * @name ViewfinderView
+   * @extends Backbone.View
+   * @screenshot views/maps/ViewfinderView.png
+   * @since 2.27.1
```

Comment:
Great I'll set as x.x.x.
I'm curious, do we actually find that @since tagging useful? It sounds like undue burden on an already fairly manual release process

Reply:
Yes, `@since` tags are useful to track when changes were made. It's really a very quick process to switch the `x.x.x`s to the current release version.

Reply:
I would say that's what the git commit history is for, but I'm not familiar with who is referencing these tags, so I'll leave that alone!